package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "io.sentry.sentry-unity",
+  "version": "0.0.5",
+  "displayName": "Sentry Unity",
+  "description": "Sentry SDK for Unity",
+  "unity": "2020.1",
```

Comment:
FYI, this is probably higher than in needs to be, but I've only tested with 2020.1 so far.

Reply:
I'll try it out on an older version then

Reply:
Lowered it to 2018.1. Tested and confirmed working on 2018.2, I don't have a 2018.1 install, though, but think it should work there as well.